module/Photo/view/photo/photo/view.phtml

```diff
@@ -5,56 +5,66 @@ $this->scriptUrl()->requireUrl('member/search');
 ?>
 <?php if ($photo != null): ?>
     <div clas="row">
+        <h1>Foto's</h1>
+        
         <nav>
             <ul class="pager">
+                <li>
+                    <a href="<?= $this->url('photo/album', array('album_id' => $photo->getAlbum()->getId())) ?>">    
+                        <?= $photo->getAlbum()->getStartDateTime()->format('Y') ?>/<?= $photo->getAlbum()->getEndDateTime()->format('Y') ?>
+                    </a>        
+                </li>
+                    
+                <li>
+                    <a href="<?= $this->url('photo/album', array('album_id' => $photo->getAlbum()->getId())) ?>">
+                        <?= $photo->getAlbum()->getName() ?>
+                    </a>
+                </li>
+            
+<!--
                 <li class="previous"><a
                         href="<?= $this->url('photo/album', array('album_id' => $photo->getAlbum()->getId())); ?>"><span
                             aria-hidden="true">&larr;</span><?= $this->translate('Return to album') ?></a></li>
+-->
             </ul>
         </nav>
     </div>
     <div class="row">
-        <div class="col-lg-9">
+        <div class="col-lg-2">
+            <?php if (!is_null($previous)): ?>
+            <nav>
+                <a id="previous" href="<?= $this->url('photo/photo', array('photo_id' => $previous->getId())); ?>">
+                    <?= $this->translate('Previous') ?>
+                </a>
+            </nav>
+            <?php endif ?>
+        </div>
+        <div class="col-lg-8">
             <img class="img-responsive" src="<?= $this->fileUrl($photo->getLargeThumbPath()) ?>" alt=""/>
-            <?php if(!is_null($photo->getWeeklyPhoto())): ?>
-                <!-- TOOD: Make this more responsive -->
-                <div class="ribbon-container">
-                    <div class="corner-ribbon shadow">
-                        <span class="glyphicon glyphicon-certificate" aria-hidden="true"></span>
-                        <?= $photo->getWeeklyPhoto()->getWeek()->format('d-m-Y') ?>
-                    </div>
-                </div>
+        </div>
```

Comment:
It appears @armandduijn removed it in b914130, probably because he didn't know what its purpose was?